extensions/google-drive/src/index.tsx

```diff
@@ -140,25 +164,27 @@ export default function Command() {
                     content={escapePath(dirname(file.displayPath))}
                     shortcut={{ modifiers: ["cmd", "shift"], key: "d" }}
                   />
-                  <Action
-                    title="Reindex Files Cache"
-                    icon={Icon.Hammer}
-                    onAction={reindexFiles}
-                    shortcut={{ modifiers: ["cmd", "shift"], key: "r" }}
-                  />
-                  <Action
-                    title="Clear File Previews Cache"
-                    icon={Icon.Trash}
-                    onAction={clearFilePreviewsCache}
-                    shortcut={{ modifiers: ["ctrl", "shift"], key: "x" }}
-                  />
+                  <ReindexFilesCacheAction reindexFiles={reindexFiles} />
+                  <ClearFilePreviewsCacheAction clearFilePreviewsCache={clearFilePreviewsCache} />
                 </ActionPanel>
               }
             />
           ))}
         </List.Section>
       ) : (
-        <List.EmptyView title={isFetching ? "Fetching files, please wait..." : "No files found"} />
+        <List.EmptyView
+          title={isFetching ? "Fetching files, please wait..." : "No files found"}
+          actions={
+            <ActionPanel>
+              {!isFetching && (
+                <>
+                  <ReindexFilesCacheAction reindexFiles={reindexFiles} />
+                  <ClearFilePreviewsCacheAction clearFilePreviewsCache={clearFilePreviewsCache} />
```

Comment:
Don't think we need to pass `clearFilePreviewsCache` as a prop here since it can be called directly from the extracted out action component (as it is already imported as an external dependency).

Reply:
Thank you!

Fixed it https://github.com/raycast/extensions/pull/1541/commits/2d8e8a360b79302f43b5ef2e29c2538615cf9bf5.